Formula/geant4.rb

```diff
@@ -0,0 +1,100 @@
+class Geant4 < Formula
+  desc "Simulation toolkit for particle transport through matter"
+  homepage "http://geant4.cern.ch"
+  url "http://geant4.cern.ch/support/source/geant4.10.04.p02.tar.gz"
+  version "10.4.2"
+  sha256 "2cac09e799f2eb609a7f1e4082d3d9d3d4d9e1930a8c4f9ecdad72aad35cdf10"
+
+  depends_on "cmake" => :build
+  depends_on "xerces-c"
+  depends_on "qt"
+
+  resource "G4NDL" do
+    url "http://cern.ch/geant4-data/datasets/G4NDL.4.5.tar.gz"
```

Comment:
All `cern.ch` URLs support HTTPS, would mind updating them?

Reply:
Done. Forgot to check to see if they had secure versions, the old formula used http too.